### PR TITLE
chore: add linkedin feature to chrome extension

### PR DESCRIPTION
**Pull request by a0m0rajab**
## Description

This PR adds the explanation of the linkedin feature to docs. 
Note: I could not run this locally there was few errors in the project that needs to be solved. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

closes https://github.com/open-sauced/docs/issues/111

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @a0m0rajab. There some parts of the PR form that are incomplete. Can you work on that please?

File: docs/tools/chrome-extension/linkedin-feature.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: 'Linkedin Integration'
+sidebar_label: 'Linkedin Integration'
+keywords:
+  - Linkedin
+  - posting
+  - Linkedin project
+---
+
+You can add a repo to Linkedin as a project through the extension, the extension will get the name of the project and description filled autoamtically for you.  
+
```
Comment (CBID2) [inline docs/tools/chrome-extension/linkedin-feature.md:12]: [suggestion]: This is a good start @a0m0rajab, but some things that are missing is:

- A statement that describes the benefit 
- A Call To Action. This is the statement that encourages people to how to get started. Check out the last sentence on the explainer page about [the Highlights feature](https://docs.opensauced.pizza/features/highlights/) for inspiration.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @a0m0rajab. Your PR is of to a great start! :) I gave some suggestions that can help you improve the explainer page.

File: docs/tools/chrome-extension/linkedin-feature.md (hunk 1 above)
Comment (a0m0rajab) [reply docs/tools/chrome-extension/linkedin-feature.md:12]: Thanks, I just added that.

**Comment by a0m0rajab** [comment]
Thanks, I added an extra two sentences! Thank you for the suggestion.

**Review by CBID2 — APPROVED** [review]
Looks great to me @a0m0rajab! 😁

**Comment by a0m0rajab** [comment]
Thanks for the approval.

**Review by BekahHW — CHANGES_REQUESTED** [review]
A couple of things here:

1. Every new page needs to be added to `sidebar.js` so people can find it. 
2. I'm not sure this needs to be its own page. The Chrome Extension has a lot of pages, but I think it actually hurts discoverability bc they're all short and folks are willing to click through all of them. I think we probably need to have a page that is just chrome extension features that combines all of the things it does, but if that's not something you're up for, we can leave it as is.
3. Can you provide screenshots and directions on how to use this feature? I honestly don't know where this option is or how to use it. I think the tricky part is that we have the chrome extension functionality in the nav bar like this:

image

And then we have this:

image


But maybe I'm missing a third, bc I'm not sure where the LI feature is.

**Comment by BekahHW** [comment]
@a0m0rajab I'm going to close this for now since I don't think the Chrome extension currently has this functionality. If there's a fix in place on the ext, feel free to reopen.

**Comment by a0m0rajab** [comment]
@BekahHW sorry not sure how I missed the previous comment. 

1. I will add that to the sidebar. 
2. We can do this in a separate PR to keep this scope for LinkedIn only. 
3. Sure, will add direction. This feature only works on the repo page and would not work on other pages in GitHub. I think it should be extended to PR/issues pages as well tbh. 

An example: https://github.com/open-sauced/docs 

image

File: sidebars.js [hunk 2]
```diff
@@ -91,6 +91,12 @@ module.exports = {
                   type: "doc",
                   id: "tools/chrome-extension/viewing-insights-and-invitations",
                 },
+                {
+                  type: "file",
+                  label: "feature 7",
```
Comment (a0m0rajab) [inline sidebars.js:96]: I do not feel this is the best label for features but just went as the previous error, we can get all the features in one page in another PR.

**Comment by BekahHW** [comment]
I'm not seeing the LinkedIn button when I use the ext:

image

**Comment by BekahHW** [comment]
Since the chrome extension doesn't currently have this functionality, I'm going to close this issue.

**Pull request closed without merging**